arduino-ide-extension/src/browser/contributions/save-as-sketch.ts

```diff
@@ -58,10 +58,11 @@ export class SaveAsSketch extends SketchContribution {
       markAsRecentlyOpened,
     }: SaveAsSketch.Options = SaveAsSketch.Options.DEFAULT
   ): Promise<boolean> {
-    const [sketch, configuration] = await Promise.all([
-      this.sketchServiceClient.currentSketch(),
-      this.configService.getConfiguration(),
-    ]);
+    const configuration = this.configService.tryGetConfig();
+    if (!configuration) {
+      return false;
```

Comment:
Fix this

Reply:
https://github.com/arduino/arduino-ide/pull/1774/commits/16abc5555ed30a9ed6af7ba04a30efc30dd18f13